src/taskgraph/main.py

```diff
@@ -556,6 +558,22 @@ def load_image(args):
         sys.exit(1)
 
 
+def validate_docker():
+    try:
+        p = subprocess.run(["docker", "ps"])
+        if p.returncode != 0:
+            print(
+                "Error: Cannot connect to the Docker daemon. Is the docker daemon running?"
```

Comment:
I know I told you to add this print, but by default, the `subprocess.run` function will print the output from the command. On my system at least, `docker ps` prints out:

> Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?

So we'd be esssentially printing the same message twice. Maybe what you could do is add `capture_output=True` to the `run` call, then instead of printing this message, you would print something like:
```
print("Error connecting to Docker:\n" + proc.stderr)
```

Reply:
Sure, Added the changes. Please have a look